doc/api-extensions.md

```diff
@@ -2273,3 +2273,6 @@ This allows copying storage volume snapshots to and from remotes.
 ## `zfs_delegate`
 This implements a new `zfs.delegate` volume Boolean for volumes on a ZFS storage driver.
 When enabled and a suitable system is in use (requires ZFS 2.2 or higher), the ZFS dataset will be delegated to the container, allowing for its use through the `zfs` command line tool.
+
+## `documentation_config`
```

Comment:
The commit message for this chdnge is out dated

Reply:
We should change the api extension to make it more general about what the URL endpoint includes as additional content will be added over time.